lib/clearance/testing/helpers.rb

```diff
@@ -1,31 +1,14 @@
+require "clerance/testing/controller_helpers"
+
 module Clearance
   module Testing
+    # @deprecated Use Clearance::Testing::ControllerHelpers
     module Helpers
-      def setup_controller_request_and_response
-        super
-        @request.env[:clearance] = Clearance::Session.new(@request.env)
-      end
-
-      def sign_in
-        unless defined?(FactoryGirl)
-          raise(
-            RuntimeError,
-            "Clearance's `sign_in` helper requires factory_girl"
-          )
-        end
-
-        factory = Clearance.configuration.user_model.to_s.underscore.to_sym
-        sign_in_as FactoryGirl.create(factory)
-      end
-
-      def sign_in_as(user)
-        @controller.sign_in user
-        user
-      end
-
-      def sign_out
-        @controller.sign_out
-      end
+      warn(
```

Comment:
> I don't expext users to see this deprecation. If they do, it's likely that they'll want to know both what to require instead AND what they should be including once they have done so.

Yes, that was my thought too. Given that they likely won't see this, I don't know how fancy this message needs to be, but I think adding "require clearance/testing/controller_helpers instead" would help them.